recipes/napari-annotatorj/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "napari-annotatorj" %}
+{% set version = "0.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-annotatorj-{{ version }}.tar.gz
+  sha256: 4e841264004c8e732f10e4f8792631d7cd39d141ea8b987d92f128da54f2ee52
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - imagecodecs
+    - keras
+    - napari-plugin-engine >=0.1.4
+    - numpy
+    - opencv >=4.5.5
+    - python >=3.7
+    - roifile
+    - scikit-image
+    - tensorflow >=2.5.0
+    - tifffile
+    - qtpy
+    - napari
+    - magicgui
```

Comment:
I cannot find this upstream, is this required?

Reply:
Do you have a link to the error? I'm just trying to make sure that it's because you're depending on it and not one of your dependencies which might mean that they are incorrectly built on conda-forge.